app/assets/javascripts/components/student/pages/lessons_page.es6.jsx

```diff
@@ -1,12 +1,35 @@
 class LessonsPage extends React.Component {
 
+  static get propTypes() {
+    return {
+      studentId: React.PropTypes.number.isRequired,
+    }
+  }
+
   constructor() {
     super();
     this.state = {
       filter: "upcoming",
+      lessons: null,
     };
   }
 
+  componentDidMount() {
+    this.fetchLessons();
+  }
+
+  fetchLessons() {
+    const {studentId} = this.props
+    const route = ApiConstants.students.upcoming_lessons(studentId);
```

Comment:
if we're going to refactor it into a `const` later on anyway, why not just teach our developers the difference between a var and const now, and keep this a `const` here?